reference/configuration/assetic.rst

```diff
@@ -1,122 +1,17 @@
 .. index::
     pair: Assetic; Configuration reference
 
-AsseticBundle Configuration ("assetic")
-=======================================
+Assetic Configuration Reference (AsseticBundle)
+===============================================
 
 .. include:: /assetic/_standard_edition_warning.rst.inc
 
-Full Default Configuration
---------------------------
+You can get the full Asstic configuration reference as follows:
 
-.. configuration-block::
+.. code-block:: terminal
 
-    .. code-block:: yaml
+    # displays the default config values defined by Symfony
+    $ php bin/console config:dump assetic
```

Comment:
I think we should write full command name: php `bin/console config:dump-references assetic`

Reply:
You are right. I've changed this. Thanks!